.github/workflows/frontend.yml

```diff
@@ -178,7 +178,7 @@ jobs:
     steps:
       -
         name: GitHub Release
-        uses: softprops/action-gh-release@7b4da11513bf3f43f9999e90eabced41ab8bb048  # v2.2.0
+        uses: softprops/action-gh-release@67b6c38432c171b8aa64dc6f3f152d12da2d99fa  # v2.2.0
```

Comment:
https://github.com/softprops/action-gh-release/commit/67b6c38432c171b8aa64dc6f3f152d12da2d99fa is not 2.2.1 but comes from master

Reply:
fixed in https://github.com/moby/buildkit/pull/5662/commits/cdabc2f4809f4b41d9653a50d7ee31c9d2e9c3ea